lcls-twincat-pmps/PMPS/HelperFunctions/F_SafeBPCompare.TcPOU

```diff
@@ -45,7 +48,7 @@ FOR idx:=1 to PMPS_GVL.MAX_APERTURES DO
     BeamParam1.astApertures[idx].Width > BeamParam2.astApertures[idx].Width;
 END_FOR
 
-F_SafeBPCompare := xAttOK AND xEvOK AND xRateOK AND xAuxAprtOK AND xAuxAttOK;
+F_SafeBPCompare := xAttOK AND xEvOK AND xRateOK AND xAuxAprtOK AND xAuxAttOK AND xBCOK;
```

Comment:
Random thought: in the event that both rate and bc were ok, could we ignore the machine mode being invalid?
Might be unnecessarily complicated but it's a thought

Reply:
Same for the zero rate compare version